Documentation/tutorials/edge-bridge-tutorial.md

```diff
@@ -242,172 +254,220 @@ edgebridgetutorialapp://
+
+<img src="../assets/edge-bridge-tutorial/data-prep/datastreams-calculated-field-set-function.png" alt="Datastream create calculated field" width="1100"/>  
+
+1. Set the editor cursor between the parentheses of the `aa_get_product_names()` function in the editor area (**1**).
+
+<img src="../assets/edge-bridge-tutorial/data-prep/datastreams-calculated-field-pre-set-field.png" alt="Datastream create calculated field" width="1100"/>  
+
+1. Select the plus button (**+**) next to **data.__adobe.analytics.products** (**1**) - it may appear truncated like: `data.__adobe.a...products`
+   1. Notice the green checkmark (**2**) in the top right of the editor area indicating correct syntax.
+   2. Select **Preview** (**3**) to see what the output of the function will be, based on what is currently in the editor and the original source JSON.
+
+<img src="../assets/edge-bridge-tutorial/data-prep/datastreams-calculated-field-set-field.png" alt="Datastream create calculated field" width="1100"/>  
+
+7. Select the schema icon (**1**) to open the XDM property viewer window.
+
+<img src="../assets/edge-bridge-tutorial/data-prep/datastreams-mapping-schema-button.png" alt="Datastream create calculated field" width="1100"/>  
+
+7. In the XDM property viewer window, select the dropdown chevron next to **productListItems** (**1**).
+8. Then choose the **name** property (**2**) and select **Select** (**3**).
+
+<img src="../assets/edge-bridge-tutorial/data-prep/datastreams-mapping-calculated-xdm-property.png" alt="Datastream create calculated field" width="1100"/>  
+
+> [!TIP] 
+> If you want to map an array of objects in the source field to an array of different objects in the target field, add `[*]` after the array name in the destination field paths.
 
-| JSON Property                           | XDM Property                       | trackAction        | trackState         |
-| --------------------------------------- | ---------------------------------- | ------------------ | ------------------ |
-| data.contextdata.product\\.add\\.event  | commerce.productListAdds.value     | :white_check_mark: |                    |
-| data.contextdata.product\\.view\\.event | commerce.productListViews.value    |                    | :white_check_mark: |
-| data.contextdata.product\\.id           | productListItems.SKU               | :white_check_mark: | :white_check_mark: |
-| data.contextdata.product\\.name         | productListItems.name              | :white_check_mark: | :white_check_mark: |
-| data.contextdata.product\\.units        | productListItems.quantity          | :white_check_mark: |                    |
+<img src="../assets/edge-bridge-tutorial/data-prep/datastreams-mapping-target-property-array.png" alt="Datastream create calculated field" width="1100"/>  
 
-12. After completing all the mappings, click **Save**.
+After creating the desired mappings, select **Save** in the top right.
```

Comment:
For this section can you add a link to full list of analytics functions available